frontend/apps/reader/modules/readersearch.lua

```diff
@@ -81,6 +83,13 @@ SRELL_ERROR_CODES[110] = _("No preceding expression in repetition.")
 SRELL_ERROR_CODES[111] = _("Expression too complex, some hits will not be shown.")
 SRELL_ERROR_CODES[666] = _("Expression may lead to an extremely long search time.")
 
+function ReaderSearch:registerKeyEvents()
+    if Device:hasKeyboard() then
+        self.key_events.ShowFulltextSearchInputBlank = { { "Alt", "Shift", "S" }, event = "ShowFulltextSearchInput", args = "" }
+        self.key_events.ShowFulltextSearchInputRecent = { { "Alt", "S" }, event = "ShowFulltextSearchInput" }
```

Comment:
Wait, when did these regress to weird shortcut combinations? :-/ I don't think we want to add such things as "Alt+S".

Reply:
regress? they never did. They were originally using <kbd>shift</kbd>, it never was just <kbd>S</kbd> if that is what you are asking. I want to add an _optional_ version of `type to search` so still don't like the idea of single key shortcuts.